.github/workflows/windows.yml

```diff
@@ -295,44 +295,50 @@ jobs:
         uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
 
       - name: Build genai libs
+        shell: cmd
         run: |
-          . "${{ env.OV_INSTALL_DIR }}/setupvars.ps1"
-          cmake -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -S ./ -B ./build/
-          cmake --build ./build/ --config ${{ matrix.build-type }} --target package -j
+          call "${{ env.OV_INSTALL_DIR }}\setupvars.bat"
+          cmake -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -S .\ -B .\build
+          cmake --build .\build --config ${{ matrix.build-type }} --target package -j
 
       - name: Build and Install dependencies
+        shell: cmd
         run: |
-          . "${{ env.OV_INSTALL_DIR }}/setupvars.ps1"
-          python -m pip install ./thirdparty/openvino_tokenizers/[transformers] --find-links ${env:OV_INSTALL_DIR}/tools
-          python -m pip install --upgrade-strategy eager -r ./samples/requirements.txt --find-links ${env:OV_INSTALL_DIR}/tools
+          call "${{ env.OV_INSTALL_DIR }}\setupvars.bat"
+          python -m pip install .\thirdparty\openvino_tokenizers\[transformers] --find-links ${env:OV_INSTALL_DIR}\tools
+          python -m pip install --upgrade-strategy eager -r .\samples\requirements.txt --find-links ${env:OV_INSTALL_DIR}\tools
           optimum-cli export openvino --trust-remote-code --weight-format fp16 --model TinyLlama/TinyLlama-1.1B-Chat-v1.0 TinyLlama-1.1B-Chat-v1.0
 
       - name: Install samples
+        shell: cmd
         run: |
-          . "${{ env.OV_INSTALL_DIR }}/setupvars.ps1"
-          cmake --install ./build/ --config ${{ matrix.build-type }} --prefix ${{ env.OV_INSTALL_DIR }}
+          call "${{ env.OV_INSTALL_DIR }}\setupvars.bat"
```

Comment:
why did you revert the powershell?

Reply:
There is no output for Test C++ samples (greedy_causal_lm) step on Windows, but it exists for other OSes.
The suggestion was that there might be some linking issues on Windows but PowerShell command succeeds or does not print anything. I tried to switch to cmd to check it.